katello_certs_tools/fileutils.py

```diff
@@ -163,19 +160,19 @@ def rhn_popen(cmd, progressCallback=None, bufferSize=16384, outputLog=None):
         progressCallback --> progress bar twiddler
         outputLog --> optional log file file object write method
     """
-    cmd_is_list = type(cmd) in (types.ListType, types.TupleType)
+    cmd_is_list = isinstance(cmd, list) or isinstance(cmd, tuple)
```

Comment:
funky, given the fact that `subprocess.Popen` will happily take a list or a tuple :)